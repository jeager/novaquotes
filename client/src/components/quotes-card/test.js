import React from 'react'
import { QuoteCard } from './'
import renderer from 'react-test-renderer'

const quote = {
  'id': 0,
  'user': {
    'name': 'Larsen Reilly',
    'photo': 'http://placehold.it/32x32'
  }
}

test('Should render correctly', () => {
  const component = renderer.create(
    <QuoteCard key={quote.id} quote={quote} index={1}/>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})