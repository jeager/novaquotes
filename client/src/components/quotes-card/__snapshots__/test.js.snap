// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className="animated flipInY card odd"
>
  <div
    className="card-author"
  >
    <h4
      className="card-author-name"
    >
      Larsen Reilly
    </h4>
  </div>
  <div
    className="card-description"
  >
    <p />
  </div>
  <div
    className="card-photo-container"
  >
    <img
      className="card-photo"
      src="http://placehold.it/32x32"
    />
  </div>
</div>
`;
